main.go

```diff
@@ -35,7 +37,15 @@ func main() {
 	if err != nil {
 		log.Fatalf("govvv: cannot get working directory: %v", err)
 	}
-	vals, err := GetFlags(wd)
+	pkg := defaultPackage
+	args = normalizeArg(args, flPackage)
+	if idx := findArg(args, flPackage); idx != -1 {
+		pkg = strings.Split(args[idx], "=")[1]
+		// remove...can't pass through to go tool
+		args = append(args[:idx], args[idx+1:]...)
```

Comment:
maybe add this logic to `goToolDryRunCmd` 

https://github.com/maguro/govvv/blob/1959ea36b5763060450cd0fcf9a6b50cda1ca64a/main.go#L95

(or possibly extract into a new method that filters such flags that shouldn't be passed to the go tool, this can be added like `strings.HasPrefix(v, flPackage+"=")`. Maybe this will yield in less hacky code than even existing one.

I assume you'll add -version soon as well, which would also go in such an argument skip list.